src/src/jwt.ts

```diff
@@ -22,21 +21,19 @@ const verify = function (req, res, next) {
     if (!detachedSignature) {
         console.error("detachedSignature is falsy")
         next()
-        //throw expection;
+        //throw expection or what to do in situation like this?
```

Comment:
In this case request is unsigned so we should return error to the caller - like 
```
res.status(401).send({
  "responseHeader": {
    "requestId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
    "sendDate": "2019-06-17T12:21:26.014Z",
    "isCallback": true
  },
  "code": "string",
  "message": "string"
}
return;
```
Just fix the values in the above example.